make/agents/svc/deploy.mk

```diff
@@ -14,6 +14,10 @@ install: manifests kustomize ## Install CRDs into the K8s cluster specified in ~
 uninstall: manifests kustomize ## Uninstall CRDs from the K8s cluster specified in ~/.kube/config. Call with ignore-not-found=true to ignore resource not found errors during deletion.
 	$(KUSTOMIZE) build $(KUSTOMIZE_ARGS) $(KUSTOMIZE_OVERLAY) | kubectl delete --ignore-not-found=$(ignore-not-found) -f -
 
+.PHONY: deploy-rbac
+deploy-rbac:
+	$(KUSTOMIZE) build config/agents/svc/rbac | kubectl apply -n $(NAMESPACE) -f -
+
```

Comment:
I'd like to keep operator installation as unified as possible.  Why are we splitting rbac rules out?  Is there any reason why `make agentsvc deploy` doesn't work in this case?

Reply:
the deployment should be pushed from Primaza's ClusterEnvironment controller when the namespace is added to the `ClusterEnvironment.serviceNamespaces`. The deployment of RBAC will be eventually implemented in the `create namespace` command of `primazactl`.